### ISSUE TITLE
Get updated palette from Bonnie

### ISSUE THREAD

**Issue by nonprofittechy** [nonprofittechy/docassemble-HousingCodeChecklist#273]
We talked about some new more vibrant colors. May want to use those along with the Bootstrap alerts

**Comment by miabonardi** [comment]
Blue: #1A73E8
Light Blue: #d1d7ed
 
Yellow: #F8C007
Light Yellow: #F8C007B0
Lighter Yellow: #F8C00780
 
Green: #7EA560
Light Green: #C7E6B0
 
Red: #CB3402

**Comment by nonprofittechy** [comment]
Can we fill in the blanks so it makes a full Bootstrap color palette? I use this site for that pretty often:

https://huemint.com/bootstrap-basic/

We just need $blue, then the main named colors (primary, secondary, etc that you see on Huemint )

**Comment by miabonardi** [comment]
```
$blue: #1A73E8;

$theme-colors: (
    "light":      #d1d7ed,
    "dark":       #7ea560,
    "primary":    #1a73e8,
    "secondary":  #f8c007,
    "info":       #c7e6b0,
    "success":    #02d178,
    "warning":    #f0d702,
    "danger":     #ff2e26,
);
```

**Comment by nonprofittechy** [comment]
Fixed in 45eaac6
